guides/index_template.md

```diff
@@ -0,0 +1,182 @@
+# Index Template
+Index templates are a convenient way to define settings, mappings, and aliases for one or more indices when they are created. In this guide, you'll learn how to create an index template and apply it to an index.
+
+## Setup
+
+Assuming you have OpenSearch running locally on port 9200, you can create a client instance
+with the following code:
```

Comment:
No need to line-wrap in markdown.

Reply:
I have removed the line-wrap from the MD file, and pushed the changes.